### ISSUE TITLE
Executing bang scripts fail on Linux

### ISSUE THREAD

**Issue by EricSeynaeve** [bangsh/bangsh#9]
When executing bang on Linux, I get the following:

```
$ cat first_bang
#!/usr/bin/env bang run

[ -n "$1" ] && b.task.run "$@"
```

```
$ ./first_bang 
/usr/bin/env: bang run: No such file or directory
```

I can quickly solve this by creating following script `bang_run` in my path:

``` bash
#!/usr/bin/env bash

bang run "$@"
```

and changing the shebang in `first_bang` to:

```
#!/usr/bin/env bang_run
```

**Comment by bellthoven** [comment]
Hey!

Have you tried creating an symbolic link of `bang` in your `/usr/local/bin` path? maybe it solves the problem. You can check the `Makefile` in the install section. Btw, can you provide the name and version of your linux distribution?

**Comment by EricSeynaeve** [comment]
Hello bellthoven,

I created a link in `/usr/bin` but no gain. The error remains.

I did find an ugly workaround by creating the wrapper script indicated above as `"/usr/bin/bang run"`. Yes, a space in an application name !

My distro is Fedora 21 (workstation edition but with KDE).

Regards,

Eric

**Comment by bellthoven** [comment]
I think this issue was closed by #10, so I am closing it.
